### PR TITLE
Feat/bottom nav bar

### LINKED ISSUE #11: Add BottomNavigationBar

### PR DESCRIPTION
**Pull request by riyan-dcosta**
fix #11

### CODE REVIEW

**Review by AslamThachapalli — APPROVED** [review]

File: lib/features/home_page/presentation/pages/home_page.dart [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
+        title: const Text(_appBarTitle),
+      ),
+      body: Center(
```
Comment (AslamThachapalli) [inline lib/features/home_page/presentation/pages/home_page.dart:30]: Add a comment here.

**Review by AslamThachapalli — CHANGES_REQUESTED** [review]

File: lib/features/home_page/presentation/pages/home_page.dart (hunk 1 above)
Comment (riyan-dcosta) [reply lib/features/home_page/presentation/pages/home_page.dart:30]: not required for now.

**Review by riyan-dcosta — COMMENTED** [review]
done

File: lib/features/profile_page/presentation/pages/profile_page.dart [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  @override
+  State<ProfilePage> createState() => _ProfilePageState();
+}
+
+class _ProfilePageState extends State<ProfilePage> {
+  static const _appTitle = "Profile Page";
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```
Comment (AslamThachapalli) [inline lib/features/profile_page/presentation/pages/profile_page.dart:15]: Add another comment here.

**Review by AslamThachapalli — CHANGES_REQUESTED** [review]

**Review by AslamThachapalli — APPROVED** [review]

**Pull request merged**